tkg/client/delete_region.go

```diff
@@ -468,7 +483,7 @@ func (c *TkgClient) cleanUpAVIResourcesInManagementCluster(regionalClusterClient
 	log.Info("Cleaning up AVI Resources...")
 	akoAddonSecretData := akoAddonSecret.Data["values.yaml"]
 	var values map[string]interface{}
-	err = yaml.Unmarshal(akoAddonSecretData, &values)
+	err := yaml.Unmarshal(akoAddonSecretData, &values)
```

Comment:
You can remove the ":"

Reply:
Refactor the code and move the step to get `isClusterClassBased` out of `cleanUpAVIResourcesInManagementCluster` function. I might still need this initiation.